### PR TITLE
docs(contributing) some fixes

### PR DESCRIPTION
**Pull request by salazar**
### Summary

Fix logic error and some English issues in CONTRIBUTING.md.

### Full changelog

* Fix logic mistake in Lua style example
* Some English cleanup

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -706,7 +706,7 @@ end
 
 -- good
 local function check_name(name)
-  if #name < 3 and #name > 30 then
```
Comment (thibaultcha) [inline CONTRIBUTING.md]: That was a test 🙄

Comment (thibaultcha) [inline CONTRIBUTING.md]: You pass

**Comment by p0pr0ck5** [comment]
LGTM. @coopr, any thoughts on the grammar cleanup portion?

**Comment by coopr** [comment]
LGTM!

**Comment by thibaultcha** [comment]
Should have probably been squashed, but hey 🍾

**Comment by p0pr0ck5** [comment]
Disagree, two separate scopes :)

**Comment by thibaultcha** [comment]
Disagree, this is docs, not code :) Doesn't matter much and pollutes the commit history more than necessary.

**Pull request merged**